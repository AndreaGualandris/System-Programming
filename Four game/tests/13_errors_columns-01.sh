#!/bin/bash
exec "$1" 7 5 o <<EOF
2 2 2 2 2 2
EOF
