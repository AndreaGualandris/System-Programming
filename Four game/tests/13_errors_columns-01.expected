-------
       
       
       
       
       
-------
0123456
Player 'o' to move:
  o
  v
-------
       
       
       
       
  o    
-------
0123456
Player 'x' to move:
  x
  v
-------
       
       
       
  x    
  o    
-------
0123456
Player 'o' to move:
  o
  v
-------
       
       
  o    
  x    
  o    
-------
0123456
Player 'x' to move:
  x
  v
-------
       
  x    
  o    
  x    
  o    
-------
0123456
Player 'o' to move:
  o
  v
-------
  o    
  x    
  o    
  x    
  o    
-------
0123456
Player 'x' to move:
Illegal move.
